modular_skyrat/master_files/code/modules/client/preferences_savefile.dm

```diff
@@ -1,4 +1,22 @@
+/**
+ * This is a cheap replica of the standard savefile version, only used for characters for now.
+ * You can't really use the non-modular version, least you eventually want asinine merge
+ * conflicts and/or potentially disastrous issues to arise, so here's your own.
+ */
+#define MODULAR_SAVEFILE_VERSION_MAX 1
+
+/datum/preferences/proc/savefile_needs_update_skyrat(savefile/S)
+	var/savefile_version
+	READ_FILE(S["modular_version"], savefile_version)
+
+	if(savefile_version < MODULAR_SAVEFILE_VERSION_MAX)
+		return savefile_version
+	return -1
+
 /datum/preferences/proc/load_character_skyrat(savefile/S)
```

Comment:
Since you're here, yeet that single-letter variable into oblivion and give it a better name.